.github/workflows/push_rockspec.yml

```diff
@@ -14,9 +14,19 @@ env:
   ROCK_NAME: "checks"
 
 jobs:
+  # Run not only on tags, otherwise dependent job will skip.
+  version-check:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check module version
+        uses: tarantool/actions/check-module-version@igrishnov/gh-204-add-hardcoded-versioning-support
```

Comment:
Unfortunately, we'll need to rework this action a bit for modules like `checks` and future `metrics` (after embedding).

Plain `require('checks')` will return embedded checks for master Tarantool, so any changes from repo will be ignored. I'll workaround these things a bit later, I think you shouldn't fix them right now (but `checks` PR should wait a bit).